npe2/manifest/contributions/_keybindings.py

```diff
@@ -1,22 +1,22 @@
 from typing import Optional
 
-from pydantic import BaseModel
 from pydantic.fields import Field
 
 from ..utils import Executable
 
 
-# the existence of the command is not validated at registration-time,
-# but rather at call time... (since commands from other extensions can be called)
-class KeyBindingContribution(BaseModel, Executable):
+# TODO: uncomment code when corresponding features enabled in napari
+class KeyBindingContribution(Executable):
     command: str = Field(
         description="Identifier of the command to run when keybinding is triggered."
     )
+    # the existence of the command is not validated at registration-time,
+    # but rather at call time... (since commands from other extensions can be called)
     key: str = Field(
         description="Key or key sequence (separate keys with plus-sign and sequences "
-        "with space, e.g. Ctrl+O and Ctrl+L L for a chord)."
+        "with space, e.g. Ctrl+O"  # and Ctrl+L L for a chord)."
```

Comment:
```suggestion
        "with space, e.g. Ctrl+O"
```

Reply:
Hmm I actually think this comment should say more 😆 . I would say like
`separate simultaneous key presses with a plus-sign e.g. Ctrl+O and sequences with space e.g. Ctrl+L L`

Reply:
Oh I like that a lot @DragaDoncila!